server/src/main/java/org/apache/cassandra/sidecar/utils/RequestUtils.java

```diff
@@ -61,4 +61,14 @@ public static Integer parseIntegerQueryParam(HttpServerRequest request, String p
         }
         return defaultValue;
     }
+
+    public static String parseStringQueryParam(HttpServerRequest request, String paramName, String defaultValue)
```

Comment:
nit: You are technically getting the param, not parsing it. Consider renaming the method